src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -112,10 +295,123 @@ class CreateProject extends React.Component {
           : projectName,
     }));
 
-  render() {
+  renderProjectForm = (projectName, team, teams, platform, organization, inFlight) => {
```

Comment:
Consider defining this as a nullary function inside render body and use variable capture to get the parameters

Reply:
Having explicit parameters may improve readability.